suite-native/module-add-accounts/src/screens/SelectAccountTypeScreen.tsx

```diff
@@ -33,8 +31,8 @@ const bulletsForKeyPath = (keyPath: TxKeyPath) => (
         id={keyPath}
         values={{
             li: chunks =>
-                chunks.map(row => (
-                    <Box flexDirection="row">
+                chunks.map((row, index) => (
+                    <Box key={`${keyPath}-${index}`} flexDirection="row">
```

Comment:
Now I realized we already have an `BulletLisItem` atom. Use it here please.

Reply:
👍  good point. I haven't noticed this

Reply:
09fa3a9df3